insta2/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+// import './CommentSection.css'
+import PropTypes from 'prop-types';
+import CommentForm from '../CommentForm';
+
+class CommentSection extends React.Component {
+   constructor(props) {
+       super(props)
+       console.log(props)
+       this.state = {
+           comments: props.comments
+        }
+    }
+    addNewComment = (event, indexNum, commentText) => {
+        console.log(this.state)
```

Comment:
you may want to console.log each of the parameters you are calling in hte function to insure that they are what you think they are. this method may not be doing what you think it is: console.log(event, indexNum, commentText), also note that you will need to event.preventDefault() to stop form from refreshing page and losing state